experimental/examples/bubblebench/editable-shared-tree/src/bubblebench.ts

```diff
@@ -14,30 +14,24 @@ export class Bubblebench extends DataObject {
 	}
 	private maybeTree?: ISharedTree = undefined;
 	private maybeAppState?: AppState = undefined;
-	static treeFactory = new SharedTreeFactory();
 
 	protected async initializingFirstTime() {
 		this.maybeTree = this.runtime.createChannel(
-			"unique-bubblebench-key-1337",
-			Bubblebench.treeFactory.type,
+			/* id: */ undefined,
+			new SharedTreeFactory().type,
 		) as ISharedTree;
 
 		this.initializeTree(this.maybeTree);
 
-		this.root.set("unique-bubblebench-key-1337", this.maybeTree.handle);
```

Comment:
There is only a single key inserted for the lifetime of this document, which occurs here at construction.

Reply:
In other examples we pull this ID out into a const above for reuse in initializingFromExisting.